semgrep_metrics.atd

```diff
@@ -129,9 +133,16 @@ type extension = {
 (* Misc *)
 (*****************************************************************************)
 
+type pro_features = {
+    (* The value is determined by the `--diff-depth` option, and it's utilized
+       for inter-file differential scanning. Since semgrep 1.46 *)
+    ?diffDepth <ocaml mutable>: int option;
+}
+
 type misc = {
     (* coupling: features is commented a lot in semgrep/PRIVACY.md *)
     features <ocaml mutable>: string list;
+    ?proFeatures <ocaml mutable>: pro_features option;
```

Comment:
would be good to also add (* since 1.46 *) also here

Reply:
Done. Thanks for the reviews!